src/renderer/dx/DxRenderer.cpp

```diff
@@ -1961,6 +1961,7 @@ CATCH_RETURN()
     if (_drawingContext)
     {
         _drawingContext->forceGrayscaleAA = _ShouldForceGrayscaleAA();
+        _drawingContext->useBoldFont = textAttributes.IsBold();

```

Comment:
This is going to be the most controversial change we ever make. :smile:

Reply:
I sincerely hope the controversy will not be a blocker for this PR, because, well, we want **BOLD** 😃 

(some meme here, if I were Mike)

Reply:
"To boldly go, where no font has bolded before" perhaps?

Reply:
I don't think this will be a problem as long we're planning to add an option for it. Not necessarily in this PR, but it's probably a good idea to have the option in place before this reaches a release build.

Reply:
@j4james by “option” do you mean the “choose which font weight is bold” one, or “choose if you want bold as bright” one?

Reply:
I mean choose if you want SGR 1 to select a bold font face or not (basically a way to disable this PR). I suppose you could potentially implement that by allowing users to choose which font weight is bold, and expect them to select the normal font weight, but that seems a bit obscure. For example, in XTerm you can just uncheck the "Bold Fonts" menu, and in Konsole you uncheck the option "Draw intense colors in bold font".

There are a whole bunch of other variations people may want to configure, e.g. disabling bold-as-bright, or only enabling a bold font when used with the extended colors (i.e. the aix bright colors and the ITU 256/RGB colors). Personally I think the latter is the best default config, since it gives you the standard behaviour for the original 8 SGR colors, while still allowing bold fonts when used with extended colors, but I'd be happy with a simple option to turn this off.

Reply:
Thanks James for the explanation. I never really thought that people would not want bold font. Just like bracketed paste. I think seldom do people want to disable bracketed paste. But you are right. Bold seems like a more intrusive feature and some folks might not fancy it.

Reply:
![image](https://user-images.githubusercontent.com/18356694/124792892-108d4b00-df13-11eb-8c11-82fa541c9690.png)

<sub>I was summoned, I had to</sub>

Reply:
#10576 filed to track adding this setting alongside this release.